umbrella/router.go

```diff
@@ -58,18 +58,15 @@ type funcRoutingInfo struct {
 	Manual      string
 }
 
-// Function to compare a lowercase version of the requested slack command to a lowercase registry command
-// Probably doesn't work because you would have to check against every command in the registry and transform them
-// Also, not sure where to call it. Presumably within the first few error checks of getFuncRoutingInfo()
-func compareCmdToReg(reqCmd string, regCmd string) {
-	lowercaseReqCmd := strings.ToLower(reqCmd)
-	lowercaseRegCmd := strings.ToLower(regCmd)
-
-	if lowercaseReqCmd != lowercaseRegCmd {
-		return false
+// confirmFunctionIsRegistered is designed to eliminate case sensitivity when calling a registered function.
+// EqualFold() is used to determine that the requested function is equal to a registered function under Unicode case-folding.
```

Comment:
This documentation is redundant. strings.EqualFold is already documented under the "strings" package.